trie/print.go

```diff
@@ -20,6 +20,26 @@ import (
 	"fmt"
 )
 
+// String returns the trie stringified through pre-order traversal
+func (t *Trie) String() string {
```

Comment:
Conflicts with `print.go`, please merge. Would recommend using gotree for cleaner output

Reply:
I merged with development and I don't see any conflicts?

Reply:
Merge the functions, they do basically the same thing

Reply:
okay done